_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Multiple data sources
+parent: Discover
+nav_order: 5
+---
+
+Multiple data sources is an experimental feature released in OpenSearch 2.4. We do not recommend using it in a production environment. For updates on the feature’s progress or to leave feedback on improving the feature, see the [multiple data sources git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2448).
```

Comment:
wrong link? why it's linked to a merged PR

Reply:
Did you meant the forum link? https://forum.opensearch.org/t/feedback-experimental-feature-connect-to-external-data-sources/11144